crates/criticaltrust/src/signatures/payload.rs

```diff
@@ -597,9 +602,8 @@ mod tests {
 
     // Utilities
 
-    #[track_caller]
```

Comment:
Removing this is unfortunate, however `track_caller` is a no-op on async functions right now.